docs/guide/cli-table.md

```diff
@@ -80,8 +80,8 @@
 | `--poolOptions.vmForks.minForks <workers>` | Minimum number of processes to run tests in |
```

Comment:
We also have these options for every pool. I think they should all inherit the same behaviour then

Reply:
I implement percentage option every pool options and now all of workflows passed!